libexec/klab-prove-all

```diff
@@ -33,42 +33,36 @@ export OBLIGATIONS=$KLAB_OUT/obligations.batch
 export BATCH_LIMIT=8
 export THREADS=${THREADS:-2}
 PNAME=$(jq -r '.name' < config.json)
-if [ "$PNAME" == "null" ]; then PNAME=""; fi
+if [ "$PNAME" == "null" ];
+   echo "Please set a name in config.json"
+   exit 1
+fi
 export PNAME
```

Comment:
what does this line do?

Reply:
Meaning what is `PNAME` usde for? It's passed to the setup script, which decides whether to create a project reports dir based on its presence.

Reply:
I mean why do we need to `export PNAME` if it's already set? I'm probably just missing some bash knowledge...

Reply:
It's to make it available to child processes.